storage/ctab/src/main/java/org/openscience/cdk/io/RdfileReader.java

```diff
@@ -0,0 +1,533 @@
+package org.openscience.cdk.io;
```

Comment:
I saw you already added a task to add the missing copyright/license header.

Reply:
👍🏼